std_msgs/Header data_header
std_msgs/Float32 gpu_usage
std_msgs/Float32 usbfs_memory_mb
